.github/actions/unit-test-dotnet/action.yml

```diff
@@ -0,0 +1,32 @@
+name: .NET Unit Test
+description: .NET Test report
+
+inputs:
+  DOTNET-VERSION:
+    required: false
+    default: "3.1.402"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: ${{ inputs.DOTNET-VERSION }}
```

Comment:
Should this be `inputs.DOTNET_VERSION`? Not sure if having a dash here will cause issues. Other inputs in these artifacts consistently use underscore instead.